src/analysis/complex/upper_half_plane/basic.lean

```diff
@@ -28,12 +28,12 @@ open matrix matrix.special_linear_group
 
 open_locale classical big_operators matrix_groups
 
-local attribute [instance] fintype.card_fin_even
-
 /- Disable this instances as it is not the simp-normal form, and having them disabled ensures
 we state lemmas in this file without spurious `coe_fn` terms. -/
 local attribute [-instance] matrix.special_linear_group.has_coe_to_fun
 
+local attribute [instance] fintype.card_fin_even
```

Comment:
Why move this?

Reply:
I think I accidentally deleted it and then put it back in a different place.